server/cmwell-it/src/it/scala/cmwell/it/SearchTests.scala

```diff
@@ -221,6 +221,22 @@ class SearchTests extends AsyncFunSpec with Matchers with Inspectors with Helper
       }
     }
 
+    val lastModifiedSearch = executeAfterCompletion(ingestGeonames){
+      spinCheck(100.millis,true)(Http.get(
+        uri = path,
+        queryParams = List("op" -> "search","format" -> "json","pretty" -> "","debug-info" -> "",
+          "recursive"-> "", "qp" -> ("system.lastModified<<" + System.currentTimeMillis())))){ r =>
```

Comment:
reading currentTimeMillis is a heavy operation. Using this way it will be read for each spin. Better is to have one time val outside of the spin check

Reply:
done